docs/examples/git-repo/kustomization-comp.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  name: kustomize-control-repo
+  namespace: default
+spec:
+  workload:
+    definition:
+      apiVersion: apps/v1
+      kind: Deployment # so the system know this is a deployment type workload
```

Comment:
This is not correct, maybe just skip this section for now.

/cc @wonderflow , we really need to be able to detect the workload type from component.

Reply:
Yes, but it's not high priority thing. Will allow this section to be empty.

Reply:
It's supported now